src/components/DiagramFrame/MetadataPanel/utils.ts

```diff
@@ -117,15 +119,17 @@ export function getFieldMetadata(fields: ILookmlModelExploreField[], selectionIn
 }
 
 export function getViewMetadata(viewResponse: ILookmlModelExplore, isLoading: boolean, lookmlLink: string, selectionInfo: SelectionInfoPacket) {
-  let sqlTableName = "unknown"
-  if (isLoading) {
-    sqlTableName = "Loading..."
-  } else if (viewResponse?.name === selectionInfo.name) {
+  let sqlTableName: string
```

Comment:
I suppose this is still overloading... But it's better?

Reply:
I should use the APIX pattern here where I pass back renderers... But that's a refactor for another day :D

Reply:
Can live with it for now